### PR TITLE
Billing - repair inconsistencies task

### PR DESCRIPTION
**Pull request by ClaudeROSSI**
Repair task to fix some inconsistencies 
- Invoice with no sessions 
- and sessions with no invoices

### CODE REVIEW

File: src/integration/billing/stripe/StripeBillingIntegration.ts [hunk 1]
```diff
@@ -1523,4 +1524,136 @@ export default class StripeBillingIntegration extends BillingIntegration {
     // Let's return the check results!
     return errorCodes;
   }
+
+  public async repairInvoice(billingInvoice: BillingInvoice): Promise<void> {
```
Comment (ClaudeROSSI) [inline src/integration/billing/stripe/StripeBillingIntegration.ts:1528]: @LucasBrazi06 : public method added only for repairing inconsistencies.
I'll remove it as soon as all invoices are ok!

File: src/migration/tasks/RepairInvoiceInconsistencies.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+import { ServerAction } from '../../types/Server';
+import StripeBillingIntegration from '../../integration/billing/stripe/StripeBillingIntegration';
+import Tenant from '../../types/Tenant';
+import TenantStorage from '../../storage/mongodb/TenantStorage';
+import Utils from '../../utils/Utils';
+// import moment from 'moment';
+
+const MODULE_NAME = 'RepairInvoiceInconsistencies';
+
+export default class RepairInvoiceInconsistencies extends MigrationTask {
```
Comment (LucasBrazi06) [inline src/migration/tasks/RepairInvoiceInconsistencies.ts:15]: I created a TenantMigrationTask lately ;-)
Extend from this one.

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]
Small changes

File: src/migration/tasks/RepairInvoiceInconsistencies.ts (hunk 2 above)
Comment (ClaudeROSSI) [reply src/migration/tasks/RepairInvoiceInconsistencies.ts]: done

**Pull request merged**